product-scenarios/scenarios-common/src/main/java/org/wso2/am/scenario/test/common/ScenarioTestBase.java

```diff
@@ -350,4 +358,31 @@ public String getHttpsAPIInvocationURL(String apiContext, String apiVersion, Str
         return gatewayHttpsURL + "/" + apiContext + "/" + apiVersion + apiResource;
     }
 
+    public static boolean isWebApplicationDeployed(String serviceEndpoint, String username, String password,
```

Comment:
why copy this method instead of using the existing one?

Reply:
The arguments given in this method are different. The existing method in WebAppDeploymentUtil class requires sessionCookie parameter which used in creating webappAdminClient. This implementation uses WebAppAdminClient in samples.utils. So this method can be invoked without specifying sessionCookie.